src/styles/index.scss

```diff
@@ -1,3 +1,100 @@
+:root {
+  --one-day-box-size: 100px;
+  --gap-between: 1px;
+  --padding-around: 10px;
+  --border-color: #000;
+  --background-color: #eee;
+  --text-color: #000;
+  --font-size: 30px;
+}
+
+* {
+  box-sizing: border-box;
+}
+
 body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
   margin: 0;
+  width: 100%;
+  height: 100vh;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: var(--font-size);
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: var(--gap-between);
+  padding: var(--padding-around);
+  width: calc((100px * 7) + (1px * 6) + (10px * 2));
```

Comment:
You have hardcoded values in the `calc` function. Instead, you can use the CSS variables you have defined earlier. This will make your code more flexible and maintainable.